qutebrowser/misc/split.py

```diff
@@ -119,6 +119,7 @@ def __iter__(self):  # pragma: no mccabe
                     self.token += nextchar
             else:
                 raise AssertionError("Invalid state {!r}!".format(self.state))
+        # pylint: enable=too-many-branches,too-many-statements
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.